src/routes/Account/modules/account.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const REQUEST_ACCOUNTS = 'REQUEST_ACCOUNTS'
+export const RECIEVE_ACCOUNTS = 'RECIEVE_ACCOUNTS'
+
+// ------------------------------------
+// Actions
+// ------------------------------------
+export function requestAccounts() {
+  return {
+    type: REQUEST_ACCOUNTS
+  }
+}
+
+export function recieveAccounts(value) {
+  return {
+    type: RECIEVE_ACCOUNTS,
+    payload: value
+  }
+}
+
+export const fetchAccounts = (): Function => {
+  return (dispatch: Function): Promise => {
+    dispatch(requestAccounts());
+
+    return fetch('http://localhost:3000/api/accounts')
+      .then(data => data.text())
+      .then(text => dispatch(recieveAccounts(JSON.parse(text).records)))
```

Comment:
i before e, except after c (receive)

also, we should have a `catch` here which dispatches `receiveAccountsFailure` or something, so in case `text` is not parseable state remains okay